packages/Badge/__tests__/Badge.test.tsx

```diff
@@ -1,31 +1,32 @@
-import React from 'react';
-import expect from 'expect';
-import { shallow } from 'enzyme';
-import Badge from '../src';
+import { shallow } from "enzyme";
+import expect from "expect";
+import React from "react";
+import Badge from "../src";
 
-describe('Badge component', () => {
-  test('should render badge type link', () => {
+describe("Badge component", () => {
+  test("should render badge type link", () => {
     const wrapper = shallow(
-      <Badge type="alert" round link>
-        Ipsum
+      <Badge type="badge--pagebuilder" link>
```

Comment:
According to the source code the type is already including this "badge--"

```
  const classes = `badge badge--${type} ${withIcon} ${color}`;

```
In that case this will end up like:
```
type="badge--badge--pagebuilder"  
```

Is this correct?

Reply:
It wasn't, now is fixed